src/components/HotRepoCard.tsx

```diff
@@ -1,15 +1,18 @@
 import { useEffect, useState } from "react";
-import { RiCheckboxCircleFill } from "react-icons/ri";
-import { FaArrowAltCircleUp } from "react-icons/fa";
 import { AiOutlineStar } from "react-icons/ai";
 import { BiGitPullRequest } from "react-icons/bi";
+import { FaArrowAltCircleUp } from "react-icons/fa";
+import { RiCheckboxCircleFill } from "react-icons/ri";
 import { VscIssues } from "react-icons/vsc";
 import Skeleton from "react-loading-skeleton";
+import useRepo from "../hooks/useRepo";
+import useVotedRepos from "../hooks/useVotedRepos";
 import { getAvatarLink } from "../lib/github";
 import humanizeNumber from "../lib/humanizeNumber";
 import StackedAvatar from "./StackedAvatar";
-import useRepo from "../hooks/useRepo";
-import useVotedRepos from "../hooks/useVotedRepos";
+
+const bugReportLink =
+        "https://github.com/open-sauced/hot/issues/new?assignees=&title=fix:";
```

Comment:
```suggestion
        "https://github.com/open-sauced/hot/issues/new";
```
I think this could actually be done as using the existing template - https://github.com/open-sauced/hot/issues/new?assignees=&labels=%F0%9F%91%80+needs+triage%2C%F0%9F%90%9B+bug&template=bug_report.yml&title=Bug%3A+